src/renderer/features/player/components/center-controls.tsx

```diff
@@ -5,6 +5,7 @@ import isElectron from 'is-electron';
 import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
+import { PlaybackSelectors } from '../../../../shared/constants/playback-selectors';
```

Comment:
Can you update the import to use the absolute path similar to the others? `/@/shared/...`